ratatool.rb

```diff
@@ -1,17 +1,15 @@
 class Ratatool < Formula
   desc "Tool for random data sampling and generation"
   homepage "https://github.com/spotify/ratatool"
-  url "https://github.com/spotify/ratatool/releases/download/v0.3.1/ratatool-0.3.1.jar"
-  sha256 "45d017c5eb019790a90dcc7cb3a3c2762219eaa5719ac93988ac7d4678500e47"
-
-  bottle :unneeded
+  url "https://github.com/spotify/ratatool/releases/download/v0.3.2/ratatool-cli-0.3.2.tar.gz"
+  sha256 "e86ef7f08e82ebaa4a75c335ddf8d531c57f292e1c4fd605127c3dc38fb90618"
 
   def install
-    libexec.install "ratatool-#{version}.jar"
-    bin.write_jar_script libexec/"ratatool-#{version}.jar", "ratatool"
+      lib.install Dir["lib/*.jar"]
+      bin.install "bin/ratatool"
   end
 
   test do
-    system "#{bin}/ratatool"
+    system "#{bin}/ratatool directSampler"
   end
```

Comment:
do we need the other features here as well aside from just `directSampler`

Reply:
We could but it's just a basic sanity check that it works and in this case prints usage messages, and the more we tie to the repo the more likely we are to have to keep this in sync as well. Don't think it's enough of a benefit